src/utils/protostar_directory.py

```diff
@@ -46,24 +47,37 @@ def protostar_version(self) -> VersionType:
                 version_s = tomli.loads(file.read())["tool"]["poetry"]["version"]
                 return VersionManager.parse(version_s)
         except FileNotFoundError:
-            getLogger().warning("Couldn't read Protostar version.")
-            return VersionManager.parse("0.0.0")
+            getLogger().warning("Couldn't read Protostar version")
+            return None
```

Comment:
Maybe we should re raise error here? Output could be useful for user to debug it in some cases WDYT?

Reply:
This warning should never be visible to the user. It's more a change for us to be able to run `python ./protostar.py`.